libraries/stdlib/stdlib_ng.mtlx

```diff
@@ -195,6 +195,59 @@
     <output name="out" type="vector4" nodename="N_img_vector4" />
   </nodegraph>
 
+  <!--
+    Node: <latlongimage>
+  -->
+  <nodegraph name="NG_latlongimage" nodedef="ND_latlongimage">
+    <separate3 name="viewDirChannels" type="multioutput">
+      <input name="in" type="vector3" interfacename="viewdir" />
+    </separate3>
+    <atan2 name="angleXZ" type="float">
+      <input name="iny" type="float" nodename="viewDirChannels" output="outx" />
+      <input name="inx" type="float" nodename="viewDirChannels" output="outz" />
+    </atan2>
+    <multiply name="scaleXZ" type="float">
+      <input name="in1" type="float" nodename="angleXZ" />
+      <input name="in2" type="float" value="-0.15915494" />
+    </multiply>
+    <add name="longitude" type="float">
+      <input name="in1" type="float" nodename="scaleXZ" />
+      <input name="in2" type="float" value="0.5" />
+    </add>
+    <multiply name="rotationDegrees" type="float">
```

Comment:
This pull request looks really close to the mark, but in my local validation it seems to remove the ability to rotate the environment background using the `Light Rotation` slider in the MaterialX Viewer.

The fix might be as simple as updating this logic that updates the appropriate shader uniform in RenderPipelineGL.cpp:

https://github.com/AcademySoftwareFoundation/MaterialX/blob/main/source/MaterialXView/RenderPipelineGL.cpp#L371

Additionally, you may want to verify that the conversion between angle units (in the location of this comment) is behaving as you expect, as it's not currently being tested in the MaterialX Viewer.

Reply:
Thanks, good points! I believe to have addressed them. Although I have to admit I do not understand where the original 0.5 offset in the light rotation logic comes from, and have therefore omitted it.

Main branch
<img width="1392" alt="Screenshot 2025-06-07 at 16 32 11" src="https://github.com/user-attachments/assets/47aec955-45e5-4400-9edb-116505a3324e" />
This PR
<img width="1392" alt="Screenshot 2025-06-07 at 16 33 24" src="https://github.com/user-attachments/assets/3c4cc08a-3a9e-4bfe-ab3b-fca3f364c4a6" />